meta-frc971/recipes-frc971/configuration/frc971-dotfiles_1.0.bb

```diff
@@ -4,26 +4,115 @@ LICENSE = "MIT"
+SRC_URI[sha256sum] = "bf7ae9cad99461a92ae43e5e2ea9ebe4951d34f42d9004990d8ff6628bacb272"
 
 S = "${WORKDIR}"
 
-do_compile() {
+SYSTEMD_AUTO_ENABLE:${PN} = "enable"
+SYSTEMD_SERVICE:${PN} = "grow-rootfs jetson-clocks frc971 frc971chrt hwclock"
+
+ROOT = "${D}${base_prefix}"
+
+install_content() {
+    install -D -m"$1" "contents/$2" "${ROOT}/$2"
+    chown "$3" "${ROOT}/$2"
 }
 
 do_install() {
-   install -D -m0755 root/bashrc ${D}${base_prefix}${ROOT_HOME}/.bashrc
-   install -D -m0755 root/bashrc ${D}${base_prefix}${ROOT_HOME}/.bash_profile
-   install -D -m0755 root/trace.sh ${D}${base_prefix}${ROOT_HOME}/trace.sh
-   install -D -m0755 ${WORKDIR}/camera_overrides.isp ${D}${base_prefix}/var/nvidia/nvcam/settings/camera_overrides.isp
+    install -D -m0755 root/bashrc ${ROOT}${ROOT_HOME}/.bashrc
+    install -D -m0755 root/bashrc ${ROOT}${ROOT_HOME}/.bash_profile
+    install -D -m0755 root/trace.sh ${ROOT}${ROOT_HOME}/trace.sh
+    install -D -m0755 ${WORKDIR}/camera_overrides.isp ${ROOT}/var/nvidia/nvcam/settings/camera_overrides.isp
+
+    install_content 0644 "etc/sysctl.d/sctp.conf" "root:root"
+    install_content 0644 "etc/systemd/logind.conf" "root:root"
+    install_content 0555 "etc/bash_completion.d/aos_dump_autocomplete" "root:root"
+    install_content 0644 "etc/security/limits.d/rt.conf" "root:root"
+    install_content 0644 "etc/systemd/system/usb-mount@.service" "root:root"
+    install_content 0644 "etc/chrony/chrony.conf" "root:root"
+
+    mkdir -p -m700 ${ROOT}/root/bin
+    chown root:root ${ROOT}/root/bin
+
+    install_content 0644 "etc/systemd/system/grow-rootfs.service" "root:root"
+    install_content 0644 "etc/systemd/system/frc971.service" "root:root"
+    install_content 0644 "etc/systemd/system/frc971chrt.service" "root:root"
+    install_content 0644 "etc/systemd/system/jetson-clocks.service" "root:root"
+    install_content 0500 "root/bin/change_hostname.sh" "root:root"
+    install_content 0500 "root/bin/chrt.sh" "root:root"
+    install_content 0700 "root/trace.sh" "root:root"
+    install_content 0440 "etc/sudoers" "root:root"
+    install_content 0644 "etc/fstab" "root:root"
+    install_content 0644 "etc/modprobe.d/audio.conf" "root:root"
+    install_content 0644 "etc/modprobe.d/can.conf" "root:root"
+    install_content 0644 "var/nvidia/nvcam/settings/camera_overrides.isp" "root:root" 
+    install_content 0644 "etc/ld.so.conf.d/yocto.conf" "root:root"
+    install_content 0644 "etc/systemd/network/eth0.network" "root:root"
+    install_content 0644 "etc/systemd/network/80-cana.network" "root:root"
+    install_content 0644 "etc/systemd/network/80-canb.network" "root:root"
+    install_content 0644 "etc/systemd/network/80-canc.network" "root:root"
+    install_content 0644 "etc/udev/rules.d/nvidia.rules" "root:root"
+    install_content 0644 "etc/udev/rules.d/can.rules" "root:root"
+    install_content 0644 "lib/systemd/system/nvargus-daemon.service" "root:root"
```

Comment:
Pretty sure this should be an override for the argus daemon package instead of here.